workbench-for-google-cloud-workstations/Justfile

```
@@ -31,7 +31,13 @@ get-build-args:
 get-build-tags:
   #!/usr/bin/env bash
   echo ${IMAGE_REGISTRY_NAME}:${RSW_TAG_VERSION},\
-  ${IMAGE_REGISTRY_NAME}:latest
+  ${IMAGE_REGISTRY_NAME}:latest,\
+  us-docker.pkg.dev/posit-images/cloud-workstations/workbench:${RSW_TAG_VERSION},\
+  us-docker.pkg.dev/posit-images/cloud-workstations/workbench:latest,\
+  asia-docker.pkg.dev/posit-images/cloud-workstations/workbench:${RSW_TAG_VERSION},\
+  asia-docker.pkg.dev/posit-images/cloud-workstations/workbench:latest,\
+  europe-docker.pkg.dev/posit-images/cloud-workstations/workbench:${RSW_TAG_VERSION},\
+  europe-docker.pkg.dev/posit-images/cloud-workstations/workbench:latest
```

Since we are using bash, could we do something like
```suggestion
  ${IMAGE_REGISTRY_NAME}:latest,\
  {us,asia,europe}-docker.pkg.dev/posit-images/cloud-workstations/workbench:${RSW_TAG_VERSION},\
  {us,asia,europe}-docker.pkg.dev/posit-images/cloud-workstations/workbench:latest
```

---

It seems like it won't work that simply for us due to the formatting the GHA requires. I can look into it though.